proto/provenance/metadata/v1/tx.proto

```diff
@@ -21,6 +21,8 @@ service Msg {
   rpc WriteScope(MsgWriteScopeRequest) returns (MsgWriteScopeResponse);
   // DeleteScope deletes a scope and all associated Records, Sessions.
   rpc DeleteScope(MsgDeleteScopeRequest) returns (MsgDeleteScopeResponse);
+  // ExpireScope expires a scope and all associated Records, Sessions.
+  rpc ExpireScope(MsgExpireScopeRequest) returns (MsgExpireScopeResponse);
```

Comment:
What's the difference between `DeleteScope` and `ExpireScope`? Why do we need an `ExpireScope`?

Reply:
ExpireScope has been removed. Defaulted back to using DeleteScope msg.